byzcoin/README.md

```diff
@@ -104,6 +104,98 @@ The design of the view-change is similar to the view-change protocol in PBFT
  correct. This technique enables nodes to synchronise and replay blocks to
  compute the most up-to-date leader.
 
+## Creation of Blocks
+
+This is the path a transaction takes from the client to the block:
+
+1. A client creates a `ClientTransaction` and sends it to one or more nodes.
+ Sending it to more than one node increases the probability that at least
+ one node correctly forwards the `ClientTransaction`.
+
+2. The nodes send the `ClientTransaction` to the leader.
+
+2.a If the leader is not responding, they send the `ClientTransaction` to
+ the other nodes, indicating that the node failed to send the transaction.
+ Every node that fails to send such a transaction to the leader will start
+ requesting a viewchange.
+
+3. The leader verifies it's a new `ClientTransaction`, and then puts the
+ `ClientTransction` in a queue. As the client might have sent the
+ `ClientTransaction` to multiple nodes, the leader might receive them more
+ than once, so he has to make sure that only unique `ClientTransaction`s are
+ included in the queue.
+
+4. If no block is being verified, the leader starts a new _proposed_ block:
+
+5. The leader collects as many `ClientTransaction`s from the queue as
+ possible, assuring that the resulting proposed block is smaller than the
+ `MaxBlockSize`.
+
+6. The leader creates a proposed block and fills the header with the
+ backward-links, the timestamp, and the other information of the skipchain.
+
+7. Then the leader executes every `ClientTransaction` in order, updating the
+ temporary state of the blockchain after every execution. Valid transactions
+ are marked as `Accepted = true` and update the temporary state, while failing
+ transactions are included with `Accepted = false` and don't update the
```

Comment:
Is "included" the right word here? I assume that if they fail, they never become part of the skip chain , or do they?

edit: I understand that they are from reading later on.
I would avoid creating the ambiguity here though and simply say: "while failing transactions are marked as `Accepted = false`"

Reply:
Rewrote it - is it clearer now?